### PR TITLE
Add test for `isSwapPossible`

### LINKED ISSUE #16: Add unit test for isPossibleSwap
Add unit test for `isSwapPossible`.

### PR DESCRIPTION
**Pull request by MHassanTariq**
# Description

This PR adds test for `isSwapPossible`. This PR also segregates tests based on their success and failure cases.

Fixes #16

## Type of change

Please delete options that are not relevant.

- [X] Updated tests

# How Has This Been Tested?

All previous tests have been segregated.
Two more tests have been added for the `isSwapPossible` function. One handles its success case, while the other handles its failure cases.

- [X] `npm test` passes.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: nft-barter-core/test/nft-barter.js [hunk 1]
```diff
@@ -343,4 +335,25 @@ contract("NFT Barter", (accounts) => {
...
+      from: takerAddress,
+    });
+
+    const swapShouldNotBePossible = await tokenInstance.isSwapPossible.call(
+      swapId
+    );
+    assert.isFalse(swapShouldNotBePossible);
+  });
+
+  it("isSwapPossible - failure cases", async () => {
```
Comment (asimm241) [inline nft-barter-core/test/nft-barter.js:353]: don't understand what's happening here.

Comment (asimm241) [reply nft-barter-core/test/nft-barter.js:353]: We should also test for behavior where tokens in a swap are transferred to some other address. 
1) initiate a swap with token-1 and token-2 where addr1 owns token-1 and addr2 owns token-2
2) token-1 is transferred to add-3 OR token-2 is transferred to addr-3
3) Swap should not be possible It should be a useless swap.

**Review by asimm241 — CHANGES_REQUESTED** [review]

File: nft-barter-core/test/nft-barter.js (hunk 1 above)
Comment (MHassanTariq) [reply nft-barter-core/test/nft-barter.js:353]: Nice catch. Added test for stale test.

Comment (asimm241) [reply nft-barter-core/test/nft-barter.js:353]: It still doesn't cover the scenario I have described above. 
Please take note that in the 2nd step you have to transfer through the simple `tranferFrom` method of ERC-721.

**Review by asimm241 — APPROVED** [review]

**Pull request merged**